extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -895,6 +895,32 @@ describes.repeated(
             expect(element.getAttribute('src')).to.equal('');
           });
 
+          it('should not render if [src] has changed since the fetch was initiated', async () => {
+            const foo = doc.createElement('div');
+            const bar = doc.createElement('div');
+
+            const firstPromise = Promise.resolve({items: [foo]});
+            const secondPromise = firstPromise.then(() => ({items: [bar]}));
+
+            listMock
+              .expects('fetch_')
+              .returns(firstPromise)
+              .returns(secondPromise)
+              .twice();
+            listMock
+              .expects('scheduleRender_')
+              .withArgs([bar])
+              .returns(Promise.resolve())
+              .once();
+            listMock.expects('showFallback_').never();
+
+            element.setAttribute('src', 'https://foo.com/list.json');
+            list.layoutCallback();
```

Comment:
Maybe add a comment that you're intentionally not awaiting this promise and why.

Reply:
I hope it is slightly more clear now. I await neither of them until the end now (where I await both)